src/styles/index.scss

```diff
@@ -2,2 +1,127 @@
+$product-info-color: #616070;
+$main-color: #060b35;
+$btn-color: #00acdc;
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-inline: 19px;
+  }
+
+  &__product-info {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    margin-bottom: 16px;
+
+    .product-info-title {
+      color: $main-color;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    .product-info-text {
+      color: $product-info-color;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__stars-reviews {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    .container__stars-reviews-star {
+      display: flex;
+    }
+
+    .container__stars-reviews-star-icon {
+      width: 16px;
+      height: 16px;
+      background-size: center;
```

Comment:
The `background-size` property should have a value describing the size of the background image. For instance, you can use `cover` or `contain` or specific dimensions (e.g., `16px 16px`). It seems like there might be a misunderstanding of the property's purpose here.